#
# Description: Modify docs/percona-toolkit.pod to fix lintian
#              W: manpage-has-bad-whatis-entry
# Forwarded: https://bugs.launchpad.net/percona-toolkit/+bug/856024
# Author: Dario Minnucci <midget@debian.org>
# Last-Update: 2011-09-22
#
Index: percona-toolkit/docs/percona-toolkit.pod
===================================================================
--- percona-toolkit.orig/docs/percona-toolkit.pod	2011-09-22 00:31:02.228586786 +0200
+++ percona-toolkit/docs/percona-toolkit.pod	2011-09-22 02:37:39.106407194 +0200
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-Percona Toolkit - Advanced command-line tools used by Percona for MySQL support and consulting
+percona-toolkit - Advanced command-line tools used by Percona for MySQL support and consulting
 
 =head1 DESCRIPTION
 
